pkg/fleet/installer/service/datadog_installer.go

```diff
@@ -53,7 +53,7 @@ func SetupInstaller(ctx context.Context) (err error) {
 		return fmt.Errorf("error creating dd-agent group: %w", err)
 	}
 	if addDDAgentUser(ctx) != nil {
```

Comment:
```suggestion
	if err = addDDAgentUser(ctx); err != nil {
```

Keeping track of the error would be good. (sorry for previous comments, I misread)